src/components/Lobby/index.js

```diff
@@ -246,12 +268,17 @@ export default function Lobby() {
           <RoomsContainer>
             <RoomsLists>
               {roomsList.length &&
-                roomsList.map(({ _id, createdBy, song }) => (
-                  <Room key={_id}>
-                    <RoomName>{createdBy}</RoomName>
-                    <RoomSong>{song?.title}</RoomSong>
-                  </Room>
-                ))}
+                roomsList.map(({ _id, createdBy, song }) => {
+                  const targetUser = connectedUsers[_id];
+                  return (
+                    <Room key={_id} onClick={() => handleRoomClick(_id)}>
+                      <RoomName>{`${createdBy} ${
+                        targetUser ? Object.keys(targetUser).length : 0
+                      }/2`}</RoomName>
+                      <RoomSong>{song?.title}</RoomSong>
+                    </Room>
```

Comment:
```suggestion
                  const targetUser = connectedUsers[_id];
                  
                  return (
                    <Room key={_id} onClick={() => handleRoomClick(_id)}>
                      <RoomName>{`${createdBy} ${
                        targetUser ? Object.keys(targetUser).length : 1
                      }/2`}</RoomName>
                      <RoomSong>{song?.title}</RoomSong>
                    </Room>
```

방이 생성 될 경우 방을 만든 유저는 바로 방 안으로 들어가기 때문에 0이 아닌 `default`는 1을 주는 것이 맞아 보이네요!